packages/mp/tests/test_mp/test_validate/test_utils/test_get_project_dependency_name.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+from mp.validate.utils import get_project_dependency_name
+
+
+class TestGetProjectDependencyName:
+    def test_simple_equality_specifier(self) -> None:
+        result = get_project_dependency_name("requests==2.25.1")
```

Comment:
consider adding types in tests
```suggestion
        result: str = get_project_dependency_name("requests==2.25.1")
```